samples/ControlCatalog/App.xaml.cs

```diff
@@ -69,7 +79,8 @@ public App()
         public override void Initialize()
         {
             Styles.Insert(0, Fluent);
-            Styles.Insert(1, DataGridFluent);
+            Styles.Insert(1, ColorPickerFluent);
+            Styles.Insert(2, DataGridFluent);
```

Comment:
Length is increased here @robloo

Reply:
haha, it's late. Thanks!

Reply:
Fixed code elsewhere following this indexing